portals/ghrc/config.json

```diff
@@ -0,0 +1,20 @@
+{
+    "hasStyles": true,
+    "hasScrpits": true,
+    "logo":{
+        "id": "ghrc-logo",
+        "link": "https://ghrc.nsstc.nasa.gov/home",
+        "title": "GHRC Home"
+    },
+    "org": "GHRC",
+    "pageTitle": "GHRC Portal",
+    "parentConfig": "edsc",
+    "query": {
+        "dataCenter": "NASA/MSFC/GHRC",
+        "hasGranulesorCwic": null
```

Comment:
Another typo: should be `hasGranulesOrCwic`

Reply:
Resolved!